script.js

```diff
@@ -2,9 +2,14 @@ let allTasks = JSON.parse(localStorage.getItem('tasks')) || [];
 let valueInput = '';
```

Comment:
let allTasks = JSON.parse(localStorage.getItem('tasks')) || [];
When working with requests, you no longer need storage